src/ExceptionHandler.php

```diff
@@ -61,32 +93,52 @@ public function render($request, Exception $e)
                 );
             }
 
+            /** @var CorsService $cors */
             $cors = $this->container->make(CorsService::class);
             $cors->addActualRequestHeaders($response, $request);
         }
 
         return $response;
     }
 
+    /**
+     * Generate exception response
+     *
+     * @param $request
+     * @param Exception $e
+     * @return mixed
+     */
     private function generateExceptionResponse($request, Exception $e)
     {
         $formatters = $this->config['formatters'];
 
         // :: notation will otherwise not work for PHP <= 5.6
         $responseFactoryClass = $this->config['response_factory'];
+
         // Allow users to have a base formatter for every response.
         $response = $responseFactoryClass::make($e);
+
         foreach($formatters as $exceptionType => $formatter) {
-            if ($e instanceof $exceptionType) {
-                if (!class_exists($formatter) ||
-                    !(new ReflectionClass($formatter))->isSubclassOf(new ReflectionClass(BaseFormatter::class))) {
-                    throw new InvalidArgumentException("$formatter is not a valid formatter class.");
-                }
-
-                $formatterInstance = new $formatter($this->config, $this->debug);
-                $formatterInstance->format($response, $e, $this->reportResponses);
-                break;
+            if (!$e instanceof $exceptionType) {
```

Comment:
I think you need to do `if (!($e instanceof $exceptionType)) {`

Otherwise a good idea to refactor it like this 👍